src/control/server/server.go

```diff
@@ -225,6 +225,29 @@ func Start(log *logging.LeveledLogger, cfg *config.Server) error {
 		control.WithConfig(cliCfg),
 		control.WithClientLogger(log))
 
+	// Init management RPC subsystem.
+	mgmtSvc := newMgmtSvc(harness, membership, sysdb)
+	eventPubSub := events.NewPubSub(ctx, log)
+	defer eventPubSub.Close()
+
+	// Forward received events to management service by default.
+	eventForwarder := control.NewEventForwarder(rpcClient, cfg.AccessPoints)
+	eventPubSub.Subscribe(events.RASTypeRankStateChange, eventForwarder)
```

Comment:
Just curious what the plan is as we add more RAS events. Might be interesting to have a notation for "subscribe to all"

Reply:
yes was planning to add that in a future iteration but I can add it now

Reply:
actually I think I will add that when we have a use case to prevent inadvertent misuse